app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -75,9 +78,14 @@ class DartdocRunner implements TaskRunner {
     return false; // no race detection
   }
 
-  Future _resolveDependencies(
-      PubEnvironment pubEnv, Task task, String pkgPath) async {
-    final pr = await pubEnv.runUpgrade(pkgPath, /* isFlutter */ false);
+  Future<bool> _usesFlutter(String pkgPath) async {
+    // TODO: detect if package is Flutter
```

Comment:
I assume you're tracking this one somewhere.

Reply:
yeah, I'll do this right after this will be merged.